tests/stress/stress_test.py

```diff
@@ -85,7 +85,7 @@ class StressTestRunner:
         'couchbase:latest',
         'golang:latest',
         'iwane/numpy-matplotlib',
-        'jenkins:latest',
+        'jenkins:jenkins',
```

Comment:
Should it be `jenkins/jenkins:latest`?

Reply:
the default tag is `latest`. So it will automatically use `latest`

Reply:
No, I meant it should be `jenkins/jenkins`, right?